host-configs/LLNL/lassen-clang@upstream.cmake

```diff
@@ -13,3 +13,5 @@ set(MPI_Fortran_COMPILER /usr/tce/packages/spectrum-mpi/spectrum-mpi-rolling-rel
 
 include(${CMAKE_CURRENT_LIST_DIR}/lassen-base.cmake)
 
+# ATS
+set(ATS_ARGUMENTS "--ats jsrun_omp --ats jsrun_bind=packed"  CACHE PATH "")
```

Comment:
These seem to be the magic arguments for getting ats to run correctly on lassen gpu builds